### PR TITLE
Replaced date-fns with dayjs

### PR DESCRIPTION
**Pull request by priscilawebdev**
**Type:** Refactor 

**Description:**

I noticed that we have many rooms to improve on performance, accessibility, etc. I will open an issue for it, but today I have started replacing the date.fn library with dayjs. 

Why? 

Comparing the  size of the two libraries, the difference is very significant. Please see: 

image
image

...and although we don't use this library much, it has a considerable size in our bundle. 

As you can see while [date-fn](https://github.com/date-fns/date-fns) has 16.8  kB  Minified + Gzipped , [dayjs](https://github.com/iamkun/dayjs) has only 2.8 kB  Minified + Gzipped.

DayJs also has all the methods we use 👏😀 In this PR, I have replaced the following methods:

addHours(new Date(), limit).getTime() ->  dayjs(new Date()).add(amount, 'hour').valueOf()
formatDistanceToNow(new Date(lastUpdate)) -> dayjs(new Date(lastUpdate)).fromNow()
format(new Date(lastUpdate), TIMEFORMAT) -> dayjs(new Date(lastUpdate)).fromNow()

Please let me know what you think about :)

ps: the library is also in active development 😉

### CODE REVIEW

File: src/utils/package.test.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ describe('formatDateDistance', (): void => {
     const date2 = dateTwoMonthsAgo();
     // FIXME: we need to review this expect, fails every x time.
     // expect(formatDateDistance(date1)).toEqual('about 2 months');
-    expect(formatDateDistance(date2)).toEqual('2 months');
+    expect(formatDateDistance(date2)).toEqual('2 months ago');
```
Comment (juanpicado) [inline src/utils/package.test.ts]: does `formatDateDistance` add the `ago` by default? 😮

File: src/utils/package.ts [hunk 2]
```diff
@@ -52,11 +55,11 @@ export function formatRepository(repository: any): string | null {
 }
 
 export function formatDate(lastUpdate: string | number): string {
-  return format(new Date(lastUpdate), TIMEFORMAT);
+  return dayjs(new Date(lastUpdate)).format('DD.MM.YYYY, HH:mm:ss');
```
Comment (juanpicado) [inline src/utils/package.ts]: You can reuse `TIMEFORMAT` here, it is above 👍

Comment (priscilawebdev) [reply src/utils/package.ts]: Oh True! I forgot 🙂

File: src/utils/package.test.ts (hunk 1 above)
Comment (priscilawebdev) [reply src/utils/package.test.ts]: Yes :D

**Pull request merged**